src/layout/MainPage.tsx

```diff
@@ -17,51 +17,71 @@ const Connecting = dynamic(() => import('../components/main/Connecting'), { ssr:
 import Menu from './SideMenu';
 import Signer from '../components/ui-signer';
 import { MyAccountProvider } from '../components/utils/MyAccountContext';
-import styled from 'styled-components';
+// import styled from 'styled-components';
 import { QueueProps } from '@polkadot/ui-app/Status/types';
 import Status from '../components/main/Status';
-import { Grid } from 'semantic-ui-react';
-import TopMenu from './TopMenu';
 import { ReactiveBase } from '@appbaseio/reactivesearch';
 import { AllElasticIndexes, ElasticNodeURL } from '../components/search/ElasticConfig';
+import { Layout, Icon, Button } from 'antd';
+import Search from '../components/search/Search';
+import InputAddress from '../components/utils/InputAddress';
 
-const WrapperConnent = styled.div`
-  background: #fafafa;
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
-  height: 100%;
-  min-height: 100vh;
-  overflow-x: hidden;
-  overflow-y: auto;
-  width: 100%;
-  padding: 0 2rem;
+const { Header, Sider, Content } = Layout;
 
-  @media(max-width: 768px) {
-    padding: 0 0.5rem;
-  }
-`;
 type Props = {
   children: React.ReactNode
 };
 
-const SideMenu = (props: Props) => (
-  <ReactiveBase
+const SideMenu = (props: Props) => {
+  const [ collapsed, setCollapsed ] = useState(false);
+
+  const toggleCollapsed = () => {
+    setCollapsed(!collapsed);
+  };
+  return <ReactiveBase
     url={ElasticNodeURL}
     app={AllElasticIndexes.join(',')}
   >
-    <TopMenu />
-    <Grid>
-      <Grid.Column width={3}>
-        <Menu />
-      </Grid.Column>
-
-      <Grid.Column stretched width={9}>
-        {props.children}
-      </Grid.Column>
-    </Grid>
-  </ReactiveBase>
-);
+  <Layout style={{ minHeight: '100vh', backgroundColor: '#fafafa !important' }}>
+    <Header style={{ background: '#fff', padding: '0 1rem', borderBottom: '1px solid #ddd', position: 'fixed', zIndex: 1, width: '100%' }} className='DfHeader'>
```

Comment:
Move `style` values to DfHeader CSS.
```suggestion
    <Header className='DfHeader'>
```